octreelib/ransac/initial_points_config.py

```diff
@@ -0,0 +1 @@
+N_INITIAL_POINTS = 6
```

Comment:
It's so strange. I guess it's specific constant for specific implementation.

Reply:
Yes, this is a workaround i had to make because numba cannot initialize cuda arrays if their size is unknown at compile time. Using this variable we can change the number of initial points used for RANSAC.

An alternative solution would be creating these arrays of random indices outside of CUDA. This would allow to get rid of the workaround, but would result in a transfer overhead which may slow down the algorithm

Reply:
I mean, why do we have created separate file for this constant?

Reply:
added comments in the code about this issue